src/quo2/components/messages/author/view.cljs

```diff
@@ -9,6 +9,29 @@
 
 (def middle-dot "·")
 
+(defn display-name [{:keys [profile-name nickname ens-name text-style]}]
+  (let [ens?      (-> ens-name string/blank? not)
+        nickname? (-> nickname string/blank? not)]
+    (if ens?
+      [text/text (merge {:weight :semi-bold
+                         :size   :paragraph-2
+                         :style  (style/ens-text)}
+                        text-style)
+       ens-name]
+      [:<>
+       (when nickname?
```

Comment:
Maybe instead of using both `when` and `when-not` you could just use an `if`?

Reply:
Fixed